std/traits.d

```diff
@@ -6839,3 +6839,348 @@ unittest
     struct BoolWrapper { bool value; }
     static assert(!ifTestable!(bool, a => BoolWrapper(a)));
 }
+
+/**
+ * Detect whether $(D X) is a type or not.
+ *
+ * Returns:
+ *     `bool`
```

Comment:
You're right, I guess it's mostly muscle memory at this point. I'll fix it when I get the chance.